src/generate_conf.rs

```diff
@@ -97,6 +118,10 @@ fn extract_interfaces(network_state: &NetworkState) -> Vec<Interface> {
             logical_name: i.name().to_owned(),
             mac_address: i.base_iface().mac_address.clone(),
             interface_type: i.iface_type().to_string(),
+            connection_ids: config_files_map
+                .get(i.name())
+                .cloned()
+                .or_else(|| Some(Vec::new())),
```

Comment:
This confirms the initial concern about applying the configurations. We should do one of the following:
1. Always include the logical name in this list since a connection file will be present for it
2. Never include the logical name in the list and merge it with these connections when applying

Reply:
1) is not true, see previous comment

Reply:
I'm not saying that either of these is true, but that we should probably do one of those. However, the `ovs` example you gave can be contradicting if `ovs0.nmconnection` file is never created.

Reply:
indeed, `ovs0.nmconnection` is never created